python/feathub/registries/registry_config.py

```diff
@@ -0,0 +1,43 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from enum import Enum
+from typing import Dict, Any
+
+from feathub.common.config import ConfigDef, BaseConfig
+from feathub.common.validators import in_set
+
+
+class RegistryType(Enum):
+    LOCAL = "local"
+
+
+REGISTRY_TYPE = "registry.type"
```

Comment:
Maybe rename this as `REGISTRY_TYPE_CONFIG` to avoid confusion with `registry_type` which refers to the value of this config? This is also the approach adopted by Kafka.

Same for other variables of config names.